analyzer/src/main/kotlin/PackageManager.kt

```diff
@@ -54,9 +54,9 @@ typealias ManagedProjectFiles = Map<PackageManagerFactory, List<File>>
 typealias ProjectResults = Map<File, List<ProjectAnalyzerResult>>
 
 /**
- * A class to represent a package manager of the given [managerName] that handles dependencies for the given
- * [projectType]. The analysis of any projects and their dependencies starts in the [analysisRoot] directory using the
- * given general [analyzerConfig]. Per-repository configuration is passed in [repoConfig].
+ * A class to represent a package manager with the given [managerName] that handles projects of the given [projectType].
+ * The analysis of any projects and their dependencies starts in the [analysisRoot] directory using the given general
```

Comment:
You can also consider these out of scope, but it may make sense to address it while at it:
- Removing "general" from "given general" as it seems to not provide any info.
- "Per-repository" configuration is misleading, for mono repos where you can have multiple repository 
  configurations in the repository. proposa, say: 
  "The analysis of any projects and their dependencies starts in the [analysisRoot] directory using the given  
    `analyzerConfig` and `repoConfig`."

Reply:
I believe "general" was there exactly to make the difference between `AnalyzerConfiguration` and `RepositoryConfiguration.RepositoryAnalyzerConfiguration` more clear.

I'll try to think of a better wording.